docs/how-prysm-works/overview-technical.md

```diff
@@ -26,6 +26,6 @@ Shards themselves are similar to the current Ethereum 1.0 chain, which stores bl
 
 ![A visualisation of the Ethereum 2.0 protocol](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LRNnKRqTm4z1mzdDqDF%2F-LmSMDZylWZCvjkTTb2l%2F-LmSV6B3TY2O6o9LkvVZ%2F1*OQavLqTl-Oinw0bNPjw9Jg.png?alt=media&token=622f9cb2-02c2-4618-b73b-e69408f1e8c9)
 
-As shown above, the beacon chain runs through a distributed network of nodes known as [beacon nodes](./the-beacon-chain). Participants who want to run a beacon node and help secure the network can stake 32 ETH to have their [validator client](./validator-clients) join the overall pool of validators, whom have the responsibility of [proposing](/docs/terminology#propose) and [attesting](/docs/terminology#attest) to new blocks on the beacon chain. This deposit does not come out of nowhere however; validators transfer Ether from the [ETH1](/docs/terminology#eth1) chain to the 2.0 system through the use of a [validator deposit contract](./validator-deposit-contract).
+As shown above, the beacon chain runs through a distributed network of nodes known as [beacon nodes](/docs/how-prysm-works/the-beacon-chain). Participants who want to run a beacon node and help secure the network can stake 32 ETH to have their [validator client](/docs/how-prysm-works/validator-clients) join the overall pool of validators, whom have the responsibility of [proposing](/docs/terminology#propose) and [attesting](/docs/terminology#attest) to new blocks on the beacon chain. This deposit does not come out of nowhere however; validators transfer Ether from the [ETH1](/docs/terminology#eth1) chain to the 2.0 system through the use of a [validator deposit contract](/docs/how-prysm-works/validator-deposit-contract).
```

Comment:
`/docs/how-prysm-works/beacon-node`, `the-beacon-chain` page slug is long phased out

Reply:
good catch!